apiBase/APIRequestUtils.es6.js

```diff
@@ -77,12 +77,12 @@ export const rawSend = (apiOptions, method, path, data, kind, cb) => {
   let s = superagent[method](url);
   s.set(requestHeaders(apiOptions));
 
-  data.app = appParameter(apiOptions);
-
-  if (kind === 'form') {
-    s.type('form');
+  if (type !== 'query') {
```

Comment:
small nitpick, but can we re-order this if? `if (type === 'query') { ... } else { ... }`